frontend/src/bundles/common/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+    Checkbox as CheckboxMUI,
+    type CheckboxProps,
+    FormControlLabel,
+} from '@mui/material';
+
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = CheckboxProps & {
+    label?: string;
+    isDisabled?: boolean;
+    isDefaultChecked?: boolean;
+    isChecked?: boolean;
+    isRequired?: boolean;
+};
+
+const Checkbox: React.FC<Properties> = ({
+    label = '',
+    isChecked,
+    isDefaultChecked,
+    isDisabled = false,
+    isRequired = false,
+    value = '',
+    className = styles.primary,
+}) => {
+    const combinedClasses = getValidClassNames(className)
+        .split(' ')
+        .map((className) => {
+            return styles[className] || className;
+        })
+        .join(' ');
```

Comment:
The logic behind it is if class exist in our component style.module.scss it will apply that class from it, otherwise it will just apply pure (global) className